### ISSUE TITLE
"Check all that apply" questions 

### ISSUE THREAD

**Issue by SBleeks** [nvaccess/nvda#5802]
NVDA does not read the main question for online questionnaires that use the "check all that apply" type of question.  When tabbing through a form, it's like the question is skipped by NVDA while  the checkbox options are tabbable.  Without the question though, the associated checkbox options have no context.

WCAG 2.0 H71 offers a technique for "Check all|" form questions.  However, even with this method, the question is not read by NVDA.
Here's the link to the WCAG page for H71: Providing a description for groups of form controls using fieldset and legend elements 
 https://www.w3.org/TR/WCAG20-TECHS/H71.html

Here is a working example, (Question 9 uses a check all that apply type of question):
https://www.surveymonkey.com/r/?sm=CIcertvuELyENTBpZir4VW4alJGjJAEkmqV93c4fhzU%3D

**Comment by jcsteh** [comment]
How are you navigating to the questions; i.e. arrow keys, tab key, ...?

**Comment by SBleeks** [comment]
It’s a form, so I’m using the TAB key.  All questions within a form should be tabbable yes?

**Comment by jcsteh** [comment]
Okay. Duplicate of #3321.
